src/printer/job.js

```diff
@@ -126,3 +128,41 @@ function getFrameState(spec) {
 
   return frameState;
 }
+
+/**
+ * Returns the map canvas size in pixels based on size units and dpi given in spec
+ * @param {PrintSpec} spec
+ * @return {Array}
+ */
+function calculatePixelSize(spec) {
```

Comment:
All your code should be tested.
Add a test for this function please

Reply:
ok I added one test, thanks for your help